### PR TITLE
feat(api-client): GET domain endpoint

### PR DESCRIPTION
**Pull request by Yserz**

### CODE REVIEW

File: packages/api-client/src/account/DomainData.ts [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+ */
+
+export interface DomainData {
+  /**
+   * Full configuration endpoint address of the domain (used by mobile clients)
+   **/
+  config_json: string;
+  /**
+   * Full webapp address of the domain (used by webapp)
+   **/
```
Comment (ffflorian) [inline packages/api-client/src/account/DomainData.ts]: Comment can become a one-liner

File: packages/api-client/src/account/DomainData.ts [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+  webapp_welcome_url: string;
+  block_cloud_users: false;
```
Comment (ffflorian) [inline packages/api-client/src/account/DomainData.ts]: This is always `false`?

File: packages/api-client/src/account/DomainData.ts [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+ * You should have received a copy of the GNU General Public License
+ * along with this program. If not, see http://www.gnu.org/licenses/.
+ *
+ */
+
+export interface DomainData {
+  /**
+   * Full configuration endpoint address of the domain (used by mobile clients)
+   **/
+  config_json: string;
```
Comment (ffflorian) [inline packages/api-client/src/account/DomainData.ts]: Comment can become a one-liner

File: packages/api-client/src/account/AccountAPI.ts [hunk 4]
```diff
@@ -21,15 +21,18 @@ import {AxiosRequestConfig} from 'axios';
...
 export class AccountAPI {
   constructor(private readonly client: HttpClient) {}
 
   public static readonly URL = {
     ACTIVATE: '/activate',
+    BY_DOMAIN: 'by-domain',
     CALLS: '/calls',
     CALLS_CONFIG: 'config',
     CALLS_CONFIG_V2: 'v2',
+    CUSTOM_INSTANCE: '/custom-instance',
```
Comment (Yserz) [inline packages/api-client/src/account/AccountAPI.ts]:
```suggestion
    CUSTOM_INSTANCE: '/custom-backend',
```

File: packages/api-client/src/account/DomainData.ts [hunk 5]
```diff
@@ -0,0 +1,26 @@
...
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program. If not, see http://www.gnu.org/licenses/.
+ *
+ */
+
+export interface DomainData {
+  /** Full configuration endpoint address of the domain (used by mobile clients) **/
+  config_json: string;
```
Comment (Yserz) [inline packages/api-client/src/account/DomainData.ts]:
```suggestion
  config_json_url: string;
```

File: packages/api-client/src/account/DomainData.ts [hunk 6]
```diff
@@ -0,0 +1,26 @@
...
+  /** Full webapp address of the domain (used by webapp) **/
+  webapp_welcome_url: string;
+  block_cloud_users: boolean;
```
Comment (Yserz) [inline packages/api-client/src/account/DomainData.ts]:
```suggestion
```

**Pull request merged**